devify

```diff
@@ -11,9 +20,11 @@ export DBUS_SESSION_BUS_ADDRESS
 
 notify() {
   if [ "$status" == "add" ]; then
-    su "$user" -c "notify-send -u low -i ${icon_pack}/$2.svg -a Devify '$1' 'Connected'"
-  else
-    su "$user" -c "notify-send -u low -i ${icon_pack}/$2.svg -a Devify '$1' 'Disconnected'"
+    su "$user" -c "notify-send -u low -i ${icon_pack}/$2.svg -a Devify '$device_name' 'Connected'"
+  fi
+
+  if [ "$status" == "remove" ]; then
+    su "$user" -c "notify-send -u low -i ${icon_pack}/$2.svg -a Devify '$device_name' 'Disconnected'"
```

Comment:
Passing arguments to a function instead of using a global variable.
```suggestion
    su "$user" -c "notify-send -u low -i ${icon_pack}/$2.svg -a Devify '$1' 'Disconnected'"
```

Reply:
done